feature-libs/user/profile/components/register/register.component.ts

```diff
@@ -229,13 +229,20 @@ export class RegisterComponent implements OnInit, OnDestroy {
   }
 
   protected onRegisterUserSuccess(): void {
-    if (this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')) {
-      this.router.go({ cxRoute: 'login' });
-    } else if (
+    if (
       this.authConfigService.getOAuthFlow() ===
-      OAuthFlow.ResourceOwnerPasswordFlow
+        OAuthFlow.ResourceOwnerPasswordFlow ||
+      this.featureConfigService.isEnabled(
+        'nativeSupportForRedirectingOAuthFlows'
+      )
     ) {
-      this.router.go('login');
+      this.router.go(
+        this.featureConfigService.isEnabled(
+          'nativeSupportForRedirectingOAuthFlows'
+        )
+          ? { cxRoute: 'login' }
+          : 'login'
```

Comment:
This change is related to the enhancements we made to support oAuth flows involving a redirect.  Alternatively, a separate flag for router destinations.